### ISSUE TITLE
some questions about training

### ISSUE THREAD

**Issue by BlueCat7** [ruiyan1995/Region_Learner#1]
Dear author, thanks your nice work!Your paper said you use 16 A100 gpus and batch size is 256 for pretraining. And I find per-gpu can run 64 batch size.So I use 16 v100 gpus, and total batch size is 1024, but the training loss can't decrease. Do you meet this problem and what's your batch size per-gpu? When I use 16 batch per-gpu, total batch size is 256, it's ok. That's so strange. Look forward your reply, thanks!

**Comment by ruiyan1995** [comment]
@BlueCat7 Limited to my IO system, I put only a batch of 16 samples on each gpu. Big total-batch-size maybe requires big learning rate.

**Comment by BlueCat7** [comment]
> @BlueCat7 Limited to my IO system, I put only a batch of 16 samples on each gpu. Big total-batch-size maybe requires big learning rate.

Thank you for your quickly reply. I also try learning rate 1.6e-3 when total batch size is 1024, but the loss still can't decrease. Maybe I should check again.

**Comment by ruiyan1995** [comment]
@BlueCat7 Which config file did you use? I will check this issue.

**Comment by BlueCat7** [comment]
> @BlueCat7 Which config file did you use? I will check this issue.

configs/diet-cc-webvid2m-pt-i2k.json
But I remove the cc3m dataset, because I don't have it now. Thanks.

**Comment by BlueCat7** [comment]
Hi，I check it again and use 16gpus with 1024 batch size. Now, after I remove the empty video the loss can decrease.So I think maybe too much the empty tensor will influence the training. Thanks.

**Comment by ruiyan1995** [comment]
@BlueCat7 Yes. The empty tensor is used to replace some bad videos. But when something goes wrong with the IO system (such as an inability to load data), the losses do not decrease.

**Comment by BlueCat7** [comment]
Got it! Thanks!
